_posts/en/newsletters/2020-04-29-newsletter.md

```diff
@@ -0,0 +1,265 @@
+  *Revault*, focuses on storing funds shared between multiple parties
+  with multisig security.  The protocol allows a subset of the parties
+  to initiate a withdrawal process by getting a beacon transaction
+  confirmed; if the other parties to the vault object to the withdrawal,
+  they have the opportunity to broadcast a second transaction that
+  returns the funds to an emergency address in the vault.  If there's no
+  objection within a certain amount of time, another transaction can
+  complete the withdrawal of the funds.  Poinsot is seeking feedback on
+  the proposal.
+
+## Field report: running a Lightning node in an enterprise environment
+
+{% include articles/suredbits-enterprise-ln.md %}
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{%
+endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+FIXME:harding update to latest versions Tuesday afternoon
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 0.20.0rc1][bitcoin core 0.20.0] is a release candidate
+  for the next major version of Bitcoin Core.
+
+- [LND 0.10.0-beta.rc5][lnd 0.10.0-beta] allows testing the next major
+  version of LND.
+
+- [C-Lightning 0.8.2-rc2][c-lightning 0.8.2] is the newest release
+  candidate for the next version of C-Lightning.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Rust-Lightning][rust-lightning repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+*Note: the commits to Bitcoin Core mentioned below apply to its master
+development branch and so those changes will likely not be released
+until version 0.21, about six months after the release of the upcoming
+version 0.20.*
+
+- [Bitcoin Core #15761][] adds an `upgradewallet` RPC, doing away
+  with the old method of upgrading upon startup, thereby
+  allowing users to unlock and upgrade their wallets to
+  [Hierarchical Deterministic (HD)][Hierarchical Deterministic ref]
+  while it is loaded.  This addition is also compatible with
+  [multi-wallet][multi-wallet] as it works on the individual wallet that
+  is specified by the RPC.
+
+- [Bitcoin Core #17509][] gui: save and load PSBT FIXME:dongcarl
```

Comment:
Big week! Native Descriptor Wallets just made it in as well 🚀 https://github.com/bitcoin/bitcoin/pull/16528

Reply:
Nice!  I wouldn't usually include that this week per our [schedule](https://gist.github.com/harding/0dffdde00761262ca094219c33b8ff22), but if I find enough time today to go through the PR, maybe test it out, and write it up, I'll include it.

Reply:
I recommend we stick to the schedule and cover it next week, since this is already a long newsletter.